Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -367,8 +376,10 @@ def handle_proxy(proxy_param_name='proxy', checkbox_default_value=False, handle_
     if handle_insecure:
         if insecure_param_name is None:
             param_names = ('insecure', 'unsecure')
+        elif isinstance(insecure_param_name, str):
+            param_names = (insecure_param_name,)  # type: ignore
```

Comment:
Add the mypy exact error being ignored. See: https://xsoar.pan.dev/docs/integrations/linting#mypy